tests/test_mathematical.py

```diff
@@ -878,13 +864,6 @@ def test_sign(data, dtype):
         assert_dtype_allclose(result, expected)
 
 
-def test_sign_boolean():
-    dpnp_a = dpnp.array([True, False])
-
-    with pytest.raises(TypeError):
-        dpnp.sign(dpnp_a)
```

Comment:
the same comment here...

- Do we have similar cupy test which cover that?
- Dpctl doesn't raise an exception, but probably has to. Should we address that as dpctl issue?